workers/unity/Packages/io.improbable.gdk.buildsystem/Util/CommandlineParser.cs

```diff
@@ -79,5 +79,25 @@ internal static BuildEnvironment GetBuildEnvironment(CommandLineArgs args)
                         "Unknown build environment");
             }
         }
+
+        internal static iOSSdkVersion GetTargetIosSdk(CommandLineArgs args)
```

Comment:
I'd really like to revisit how we capitalise `iOS` at some point

Reply:
we could do it the unreal way: `IOS`